src/components/TeacherComponents/ManualEntrySchoolDetails/ManualEntrySchoolDetails.tsx

```diff
@@ -56,8 +54,9 @@ const ManualEntrySchoolDetails: FC<ManualEntrySchoolDetailsProps> = ({
             <Input
               label="School name"
               placeholder="Type school name"
-              value={value}
+              value={value ?? ""}
```

Comment:
Prevents the input from moving between a controlled and uncontrolled state when initialised without a `value`